src/plugin/regions.js

```diff
@@ -342,8 +343,9 @@ class Region {
                         : null;
 
                     e.stopPropagation();
-                    startTime =
-                        this.wavesurfer.drawer.handleEvent(e, true) * duration;
+                    startTime = this.wavesurfer.regions.util.getRegionSnapToGridValue(
```

Comment:
add comment what value `startTime` is expected to be

Reply:
startTime has not been introduced with this PR. I tried to explain it better.